docusaurus/docs/key-concepts/ui-extensions.md

```diff
@@ -23,11 +23,11 @@ The UI extensions framework is built around these concepts:
 
 - **Extension point**: A place in Grafana Core or in a plugin where content can be hooked into.
 
-- **Renderable content**: Functionality (link or component) made available to render in an extension point, or a component exposed to be used by another plugin.
+- **Renderable content**: Functionality (link, component or function) made available to render in an extension point, or a component exposed to be used by another plugin.
```

Comment:
I would probably omit `function` from here for the time being (until we only have "Renderable content" or "Registered content") as strictly speaking it's not always renderable.